client/extensions/woocommerce/app/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { canCurrentUser } from 'state/selectors';
+import config from 'config';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isSiteAutomatedTransfer } from 'state/selectors';
+import route from 'lib/route';
+
+class App extends Component {
+
+	static propTypes = {
+		canUserManageOptions: PropTypes.bool,
+		siteId: PropTypes.number,
+		currentRoute: PropTypes.string,
+		isAutomatedTransfer: PropTypes.bool,
```

Comment:
If you're wanting to firm up propTypes here, I'd recommend making several of these props required, and also adding a propType for `children`. If you want to ensure it's a single valid React Element, you can use `children: PropTypes.element.isRequired` otherwise, you can be more permissive.